src/agent/adu_core_interface/src/adu_core_interface.c

```diff
@@ -446,16 +446,17 @@ void OrchestratorUpdateCallback(
         Log_Debug("Processing deployment %s ...", workflowId);
 
         ADUC_Result inProgressResult = { .ResultCode = ADUC_GeneralResult_Success, .ExtendedResultCode = 0 };
-        if (!ReportPreDeploymentProcessingState(propertyValue, ADUCITF_State_DeploymentInProgress, workflowData, inProgressResult))
+        if (!ReportPreDeploymentProcessingState(
+                propertyValue, ADUCITF_State_DeploymentInProgress, workflowData, inProgressResult))
         {
             Log_Warn("Reporting InProgress failed. Continuing processing deployment %s", workflowId);
         }
 
         // Ensure update to latest rootkey pkg, which is required for validating the update metadata.
-        workFolder = workflow_get_workfolder(workflowData->WorkflowHandle);
+        workFolder = workflow_get_workflow_base_dir(workflowData->WorkflowHandle);
```

Comment:
what's the difference between "..._workfolder" and "..._workflow_base_dir"?  Seems like these can be (and were) easily confused?

Reply:
Understood.

The workfolder is the workfolder for individual workflows.

In this case we're building the workflow folder before we launch into the workflow to do the root key json validation. Hence why we have this function here.

This file has been changed since the last review. Can you let me know if it still looks good? Otherwise I can explicitly change the naming convention to make the goals clearer.

Reply:
still confused.

Each workflow has a unique workfolder.  In that case,  `workflow_get_workfolder()` as was previously here seems fine to me.  Is there a reason you feel that `workflow_get_workflow_base_dir` is clearer?